openedx/core/djangoapps/content/course_overviews/tests/test_course_overviews.py

```diff
@@ -789,6 +789,25 @@ def test_cdn_with_external_image(self, modulestore_type):
             self.assertTrue(modified_urls['small'].startswith(expected_cdn_url))
             self.assertEqual(modified_urls['large'], start_urls['large'])
 
+    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_cdn_with_a_single_external_image(self, modulestore_type):
+        """
+        Test that we return CDN prefixed URLs unless they're absolute.
+        """
+        with self.store.default_store(modulestore_type):
+            course = CourseFactory.create(default_store=modulestore_type)
+            overview = CourseOverview.get_from_id(course.id)
+
+            # Now enable the CDN...
+            AssetBaseUrlConfig.objects.create(enabled=True, base_url='fakecdn.edx.org')
+            expected_cdn_url = "//fakecdn.edx.org"
+
+            start_url = "/static/overview.png"
+            modified_url = overview.apply_cdn_to_url(start_url)
+
+            self.assertNotEqual(start_url, modified_url)
+            self.assertTrue(modified_url.startswith(expected_cdn_url))
```

Comment:
This test has the same docstring as the previous test and it's hard to tell the difference. It took me a while to figure out that this is testing a slightly different method. Plus it doesn't actually test what it says in the docstring, because this test case doesn't test the case where `start_url` is already an absolute URL.

Maybe combine the tests, or change the docstring to say "Test the `apply_cdn_to_url` method directly when used with a single URL"

Reply:
Oh, it seems I overlooked the docstring here. Thanks for pointing out! I'll fix this as well

Reply:
I think you forgot to address this review comment ^

Reply:
Oh, you are totally right. I'm doing it just right now. Sorry about that.